index.js

```diff
@@ -14,13 +14,18 @@ module.exports = {
 
   included: function (app) {
     this._super.included(app);
+    let addonOptions = (this.parent && this.parent.options) ||
+                       (this.app && this.app.options);
+    let config = addonOptions[this.name] || {};
+    this.markdownRoot = path.resolve(this.project.root, config.markdownRoot) ||
```

Comment:
I do not believe this will work the way you intend.  https://nodejs.org/api/path.html#path_path_resolve_paths will return the current working directory if an absolute path is not created, as well as it will prepend whatever values you throw at it.  For example, `path.resolve('not', 'a', 'real', 'path')` will return `/<cwd>/not/a/real/path`.  It also only throws errors if the arguments are not strings.  So I do not see how the `||` condition of this statement will be reached.  If the `path.resolve()` call throws an error it's an actual error, and will not make it to the `||` condition.

Reply:
A nice catch on the || - LOL. Will push an update for that when I get a chance. Should just be checking for the markdownRoot property.